PresentScreenings/ScreeningsPlanner.cs

```diff
@@ -93,32 +122,37 @@ private void DisplayResultsOfRating(FilmRating rating, string filmFan, List<Film
             // Display summary for this rating.
             int highRatedFilmCount = films.Count;
             int plannedFilmCount = films.Where(f => HasAttendedScreening(f, filmFan)).Count();
-            string fmt = "{0} out of {1} films with rating {2} planned for {3}.";
-            _builder.AppendFormat(fmt, plannedFilmCount, highRatedFilmCount, rating, filmFan);
+            string str = $"{_logTime()}  {plannedFilmCount} out of {highRatedFilmCount} films with rating {rating} planned for {filmFan}.";
+            _builder.AppendFormat(str);
```

Comment:
Please use `AppendLine()` or `Append()` instead of `AppendFormat()` because there is no longer a format.

Reply:
Done (for both cases).